package.json

```diff
@@ -25,9 +25,9 @@
     "linux"
   ],
   "sideEffects": false,
-  "main": "dist/index.js",
-  "module": "dist/index.js",
-  "jsnext:main": "dist/index.js",
+  "main": "dist/src/index.js",
```

Comment:
don't forget to change this path into size-limit 
`"size-limit": [
    {
"path": "dist/index.js",`